src/libs/actions/User.js

```diff
@@ -113,7 +113,8 @@ function getUserDetails() {
             const loginList = _.where(response.loginList, {partnerName: 'expensify.com'});
             const expensifyNewsStatus = lodashGet(response, 'account.subscribed', true);
             const validatedStatus = lodashGet(response, 'account.validated', false);
-            Onyx.merge(ONYXKEYS.USER, {loginList, expensifyNewsStatus: !!expensifyNewsStatus, validated: !!validatedStatus});
+            Onyx.merge(ONYXKEYS.USER, {expensifyNewsStatus: !!expensifyNewsStatus, validated: !!validatedStatus});
+            Onyx.set(ONYXKEYS.USER_LOGIN_LIST, loginList);
```

Comment:
Prefer merge

Reply:
Wouldn't that make our loginList grow in size with lots of repeat logins?

Reply:
Yeah, Correct. LodashMerge.....